src/ol_infrastructure/applications/mitxpro/__main__.py

```diff
@@ -128,12 +126,38 @@
 )
 mitxpro_edxapp_db = OLAmazonDB(mitxpro_edxapp_db_config)
 
+hyphenated_db_purpose = (mitxpro_config.require("db_purpose")).replace("_", "-")
+
+edx_role_statments = mysql_sql_statements.update(
+    {
+        f"edxapp-{hyphenated_db_purpose}": {
+            "create": "CREATE USER '{{{{name}}}}'@'%' IDENTIFIED BY '{{{{password}}}}';"
+            "GRANT SELECT, INSERT, UPDATE, DELETE, CREATE, INDEX, DROP, ALTER, REFERENCES"  # noqa: Q000
+            "CREATE TEMPORARY TABLES, LOCK TABLES ON {{{{ app }}}}.* TO '{{{{name}}}}'@'%';",
+            "revoke": "DROP USER '{{{{name}}}}';",
+        },
+        f"edxapp-{hyphenated_db_purpose}": {
```

Comment:
This is a duplicate of the previous statement. Is this intended to be for the CSMH database?